src/module/item/flows/ItemAvailabilityFlow.ts

```diff
@@ -34,17 +34,19 @@ export const ItemAvailabilityFlow = {
      * @param rating The rating of the technology item
      */
     prepareAvailabilityValue(availability: string, adjusted: boolean, rating: number) {
-        const availParts = ItemAvailabilityFlow.parseAvailibility(availability);
+        const availParts = availability === '0' || availability === '' ? { availability: 0, restriction: '' } : ItemAvailabilityFlow.parseAvailibility(availability);
```

Comment:
This logic should be moved into the function that's parsing availability code to its parts. Moving it outside of it defeats the point of having a simple function call.